package.json

```diff
@@ -197,6 +197,21 @@
         "filetype": "ts",
         "tokenCase": "camel"
       },
+      "colors": {
+        "exclude": {
+          "group": [
+            "Internal"
+          ],
+          "token": "\\b(Figma)\\b.*"
```

Comment:
Do we have tokens that start with "Figma"? If you just copied this from the Figmage readme example it might not be relevant for this project.

Reply:
Yes that was from the Figmage readme, I'll remove it